code/upaya/app/assets/javascripts/idp_help.js.erb

```diff
@@ -0,0 +1,16 @@
+$(document).on('page:change', function () {
+  var slideTime = 300;
+
+  $('#help-button').click(function () {
+    if ($('#help-content').hasClass('open')) {
+      $('#help-content').removeClass('open');
+      $('#help-content').slideUp(slideTime);
+      $('#help-label').html("<%= raw(I18n.t('idp.labels.where_find') + I18n.t('idp.labels.where_find_hidden')) %>");
+    } else {
+      $('#help-content').addClass('open');
+      $('#help-content').slideDown(slideTime);
+      $('#help-label').html("<%= raw(I18n.t('idp.labels.hide_finder') + I18n.t('idp.labels.hide_finder_hidden')) %>");
+      $('#help-content').removeClass('slide-panel');
+    }
+  });
+});
```

Comment:
IdProofing is hardcoded to work with IdPaaS. Do we want to import that code as part of this PR? If so, there are various missing files that are needed for IdProofing to work, such as `idp_controller` and `saml_idp_controller`. My vote would be to submit the IdProofing stuff in a separate PR to keep things small and organized. That way, if someone in the future wanted to see what files are needed for IdProofing, they can look at one small PR, rather than a huge one with non-related files.